assets/xml/objects/object_cs.xml

```diff
@@ -0,0 +1,60 @@
+        <Limb Name="gGraveYardKidLeftThighLimb" LimbType="Standard" Offset="0x845C"/>
+        <Limb Name="gGraveYardKidLeftShinLimb" LimbType="Standard" Offset="0x8468"/>
+        <Limb Name="gGraveYardKidLeftFootLimb" LimbType="Standard" Offset="0x8474"/>
+        <Limb Name="gGraveYardKidRightThighimb" LimbType="Standard" Offset="0x8480"/>
+        <Limb Name="gGraveYardKidRightShinLimb" LimbType="Standard" Offset="0x848C"/>
+        <Limb Name="gGraveYardKidRightFootLimb" LimbType="Standard" Offset="0x8498"/>
+        <Limb Name="gGraveYardKidChestLimb" LimbType="Standard" Offset="0x84A4"/>
+        <Limb Name="gGraveYardKidLeftSholderLimb" LimbType="Standard" Offset="0x84B0"/>
+        <Limb Name="gGraveYardKidLeftArmLimb" LimbType="Standard" Offset="0x84BC"/>
+        <Limb Name="gGraveYardKidLeftHandLimb" LimbType="Standard" Offset="0x84C8"/>
+        <Limb Name="gGraveYardKidRightSholderLimb" LimbType="Standard" Offset="0x84D4"/>
+        <Limb Name="gGraveYardKidRightArmLimb" LimbType="Standard" Offset="0x84E0"/>
+        <Limb Name="gGraveYardKidRightHandLimb" LimbType="Standard" Offset="0x84EC"/>
+        <Limb Name="gGraveYardKidHeadLimb" LimbType="Standard" Offset="0x84F8"/>
+
+        <DList Name="gGraveYardKidTorsoDL" Offset="0x7910"/>
+        <DList Name="gGraveYardKidLeftThighDL" Offset="0x7F00"/>
+        <DList Name="gGraveYardKidLeftShinDL" Offset="0x8050"/>
+        <DList Name="gGraveYardKidLeftFootDL" Offset="0x81E8"/>
+        <DList Name="gGraveYardKidRightThighDL" Offset="0x7A30"/>
+        <DList Name="gGraveYardKidRightShinDL" Offset="0x7B80"/>
+        <DList Name="gGraveYardKidRightFootDL" Offset="0x7D18"/>
+        <DList Name="gGraveYardKidChestDL" Offset="0x65B0"/>
+        <DList Name="gGraveYardKidLeftUpperArmDL" Offset="0x7388"/>
+        <DList Name="gGraveYardKidLeftArmDL" Offset="0x7540"/>
+        <DList Name="gGraveYardKidLeftHandDL" Offset="0x7788"/>
+        <DList Name="gGraveYardKidRightUpperArmDL" Offset="0x6CA8"/>
+        <DList Name="gGraveYardKidRightArmDL" Offset="0x6E60"/>
+        <DList Name="gGraveYardKidRightHandDL" Offset="0x70A8"/>
+        <DList Name="gGraveYardKidHeadDL" Offset="0x6808"/>
+
+        <Animation Name="gGraveYardKidWalkAnim" Offset="0x0700"/>
+        <Animation Name="gGraveYardKidSwingStickUpAnim" Offset="0x0E10"/>
+        <Animation Name="gGraveYardKidGrabStickTwoHandsAnim" Offset="0x1588"/>
+        <Animation Name="gGraveYardKidIdleAnim" Offset="0x195C"/>
+
+
+        <Texture Name="gGraveYardKidEyesOpenTex" OutName="eyes_open" Format="rgb5a1" Width="64" Height="16" Offset="0x2130"/>
+        <Texture Name="gGraveYardKidEyesHalfTex" OutName="eyes_half" Format="rgb5a1" Width="64" Height="16" Offset="0x2930"/>
+        <Texture Name="gGraveYardKidEyesClosedTex" OutName="eyes_closed" Format="rgb5a1" Width="64" Height="16" Offset="0x3130"/>
+
+        <Texture Name="gGraveYardKidTLUT" OutName="palette" Format="rgb5a1" Width="16" Height="16" Offset="0x1970"/>
```

Comment:
id add a slightly more descriptive name to the pallete if possible, and use the new tlut abbreviation with it instead of palette

Reply:
Since this is the only palette/tlut I didn't bother naming it.